internal/policy/source/source.go

```diff
@@ -144,13 +144,21 @@ func InlineData(source []byte) PolicySource {
 }
 
 func (s inlineData) GetPolicy(ctx context.Context, workDir string, showMsg bool) (string, error) {
-	dest := uniqueDestination(workDir, s.Subdir(), fmt.Sprintf("%x", (sha256.Sum256(s.source))))
+	dest := path.Join(workDir, s.Subdir())
```

Comment:
With the use of `downloadCache` the previous `dest` using `uniqueDestination` can be used

Reply:
Just tried. It doesn't work because it makes the key unique on every invocation of `GetPolicy`.